bin/civi-download-tools

```diff
@@ -367,9 +367,9 @@ function get_system_installer() {
 
 ###############################################################################
 
-function do_system_xenial() {
+function do_system_xenial() { # LTS Removal Oct 2021
   set -e
-    PACKAGES="acl git wget unzip zip mysql-server mysql-client php7.1-cli php7.1-imap php7.1-ldap php7.1-curl php7.1-mysql php7.1-intl php7.1-gd php7.1-mcrypt php7.1-bcmath php7.1-mbstring php7.1-soap php7.1-zip php7.1-xml apache2 libapache2-mod-php7.1 nodejs"
+    PACKAGES="acl git wget unzip zip mysql-server mysql-client php7.3-cli php7.3-imap php7.3-ldap php7.3-curl php7.3-mysql php7.3-intl php7.3-gd php7.3-mcrypt php7.3-bcmath php7.3-mbstring php7.3-soap php7.3-zip php7.3-xml apache2 libapache2-mod-php7.3 nodejs"
```

Comment:
php7.3-mcrypt should not exist, since it was dropped with PHP 7.2, but to my understanding support for xenial was effectively dropped in line 359 anyway?

Reply:
You're right - I'll raise a PR now to remove this. We still support Xenial until October.

Reply:
It is deprecated, but still available in the Sury repo:

```
$ apt-cache policy php7.3-mcrypt
php7.3-mcrypt:
  Installed: (none)
  Candidate: 3:1.0.4-5+0~20210223.3+debian10~1.gbp68347e
  Version table:
     3:1.0.4-5+0~20210223.3+debian10~1.gbp68347e 900
        900 https://packages.sury.org/php buster/main amd64 Packages
```

Reply:
Ah okay - then I'll leave it for now.

Reply:
We can drop it when we move BK to 7.4. Shouldn't be that long either!